src/Service/CaptureServiceImpl.cpp

```diff
@@ -193,6 +200,37 @@ static void StopInternalProducersAndCaptureStartStopListenersInParallel(
   }
 }
 
+static ProducerCaptureEvent CreateCaptureStartedEvent(const CaptureOptions& capture_options,
+                                                      uint64_t capture_start_timestamp_ns) {
+  ProducerCaptureEvent event;
+  CaptureStarted* capture_started = event.mutable_capture_started();
+
+  int32_t target_pid = capture_options.pid();
+
+  capture_started->set_process_id(target_pid);
+  auto executable_path_or_error = orbit_base::GetExecutablePath(target_pid);
+
+  if (executable_path_or_error.has_value()) {
+    const std::string& executable_path = executable_path_or_error.value();
+    capture_started->set_executable_path(executable_path);
+
+    ErrorMessageOr<std::unique_ptr<orbit_elf_utils::ElfFile>> elf_file_or_error =
+        orbit_elf_utils::ElfFile::Create(executable_path);
+    if (elf_file_or_error.has_value()) {
+      capture_started->set_executable_build_id(elf_file_or_error.value()->GetBuildId());
+    } else {
+      ERROR("Unable to load module: %s", elf_file_or_error.error().message());
```

Comment:
The module for the error is already in `elf_file_or_error.error().message()`, yes?